hz: 10                   # ループ周期 [Hz]
index_step: 5            # １回で更新するインデックス数
goal_index: 50           # グローバルパス内におけるローカルゴールのインデックス
target_dist_to_goal: 1.7 # 現在位置-ゴール間の距離 [m]
